<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Com_Network</class>
 <widget class="QWidget" name="Com_Network">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Com_Network</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_info">
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>flechazo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_Tcpip">
      <attribute name="title">
       <string>TcpIp</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_com_view">
         <property name="title">
          <string>显示</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTabWidget" name="tabWidget_config">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>160</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_config_tcpip_server">
             <attribute name="title">
              <string>TcpIp_Server</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox_config_tcpip_server_address">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>120</height>
                 </size>
                </property>
                <property name="title">
                 <string>地址配置</string>
                </property>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="1" column="4">
                  <widget class="QComboBox" name="comboBox_config_tcpip_server_type">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_config_tcpip_server_ip">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>90</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>IP:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="6" rowspan="2">
                  <widget class="QPushButton" name="pushButton_config_tcpip_server">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>60</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>侦听</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1" colspan="5">
                  <widget class="QComboBox" name="comboBox_config_tcpip_server_ip">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0" colspan="2">
                  <widget class="QLabel" name="label_config_tcpip_server_port">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>35</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>90</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>port:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="2">
                  <widget class="QLineEdit" name="lineEdit_config_tcpip_server_port">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>20</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="3">
                  <widget class="QLabel" name="label_config_tcpip_server_type">
                   <property name="text">
                    <string>type:</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_config_tcpip_client">
             <attribute name="title">
              <string>TcpIp_Client</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QGroupBox" name="groupBox_config_tcpip_client_info">
                <property name="title">
                 <string>info</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="groupBox_config_tcpip_client_ip">
                <property name="title">
                 <string>ip配置</string>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_config_tcpip_client_ip">
                   <property name="minimumSize">
                    <size>
                     <width>40</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>IP:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBox_config_tcpip_client_ip">
                   <property name="minimumSize">
                    <size>
                     <width>240</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="editable">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_config_tcpip_client_port">
                   <property name="minimumSize">
                    <size>
                     <width>40</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>port:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="lineEdit_config_tcpip_client_port">
                   <property name="minimumSize">
                    <size>
                     <width>40</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>180</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_config_tcpip_client_ip_type">
                   <property name="minimumSize">
                    <size>
                     <width>40</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>协议</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="comboBox_config_tcpip_client_type">
                   <property name="minimumSize">
                    <size>
                     <width>60</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>60</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="pushButton_config_tcpip_client">
                   <property name="minimumSize">
                    <size>
                     <width>60</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>连接</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="textBrowser_com_view"/>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_com_view_operate">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="title">
             <string>发送</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="1" colspan="7">
              <widget class="QLineEdit" name="lineEdit_com_view_operate_send"/>
             </item>
             <item row="1" column="3">
              <widget class="QLCDNumber" name="lcdNumber_com_view_operate_recvnum"/>
             </item>
             <item row="1" column="4">
              <widget class="QCheckBox" name="checkBox_com_view_operate_hexsend">
               <property name="text">
                <string>16进制发送</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QCheckBox" name="checkBox_com_view_operate_hexrecv">
               <property name="text">
                <string>16进制显示</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_com_view_operate_send">
               <property name="text">
                <string>发送数据：</string>
               </property>
              </widget>
             </item>
             <item row="0" column="9">
              <widget class="QPushButton" name="pushButton_com_view_operate_send">
               <property name="text">
                <string>发送</string>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QLCDNumber" name="lcdNumber_com_view_operate_sendnum"/>
             </item>
             <item row="1" column="9">
              <widget class="QComboBox" name="comboBox_com_view_operate_sendmode">
               <item>
                <property name="text">
                 <string>单播发送</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>广播发送</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_com_func">
         <property name="title">
          <string>操作</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_config_tcpip_server_device">
            <property name="title">
             <string>客户端</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButton_config_tcpip_server_device_add">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>90</height>
                </size>
               </property>
               <property name="text">
                <string>添加</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pushButton_config_tcpip_server_device_del">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>90</height>
                </size>
               </property>
               <property name="text">
                <string>断开</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" rowspan="2">
              <widget class="QTableWidget" name="tableWidget_config_tcpip_server_device_view">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>120</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>460</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidget_com_func">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_com_func">
             <attribute name="title">
              <string>Tab 1</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_com_func2">
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_MAC">
      <attribute name="title">
       <string>MAC</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
